docs/excel/excel-add-ins-core-concepts.md

```diff
@@ -254,5 +254,6 @@ When an API error occurs, the API will return an **error** object that contains
  
 * [Get started with Excel add-ins](excel-add-ins-get-started-overview.md)
 * [Excel add-ins code samples](https://developer.microsoft.com/office/gallery/?filterBy=Samples)
+* [Advanced Excel add-ins programming](excel-add-ins-advanced-concepts.md)
```

Comment:
link text should be changed to: Advanced programming concepts with the Excel JavaScript API